Develop

yapracticumplakhotnik opened the pull request.

Commit:
add allure report

Commit:
Diplom_3

mkouznetsov-git commented on src/test/java/tests/ConstructorTest.java:
```diff
@@ -0,0 +1,54 @@
+
+    @Test
+    @DisplayName("Раздел «Конструктор» Проверь, что работают переходы к разделам:«Булки»,«Соусы»,«Начинки».")
+    public void testConstructorSections() {
```
В этом методе 3 отдельных проверки -- надо разделить на 3 тестовых метода. Один тест должен проверять что-то одно, тесты должны быть независимыми и атомарными. Сейчас если первая проверка упадёт, то дальнейшие даже не будут выполнены.

mkouznetsov-git commented on src/test/java/tests/ConstructorTest.java:
```diff
@@ -0,0 +1,54 @@
+
+        // Проверяем переход к разделу "Булки"
+        homePage.clickSaucesSection();
+        homePage.clickBunsSection();
```
Если удалить строчки перехода по табам, то тест всё-равно проходит успешно. Значит проверка переходов по закладкам не работает.

mkouznetsov-git commented on src/test/java/tests/NavigationTest.java:
```diff
@@ -0,0 +1,73 @@
+        // Переход по клику на логотип
+        driver.get(TestData.BASE_URL + "/account");
+        accountPage.clickLogoLink();
+        assertTrue(driver.getCurrentUrl().contains("/"));
```
Это недостаточная проверка -- что URL содержит косую черту, там много таких символов, например, http://..../...
Нужна конкретная проверка.

Commit:
Diplom_3
правки после ревью: ConstructorTest.java и NavigationTest.java
тесты разделены проверки скорректированы
